src/clarity-angular/datagrid/datagrid-row.ts

```diff
@@ -20,6 +24,11 @@ import {Selection} from "./providers/selection";
     }
 })
 export class DatagridRow {
+
+    @ContentChildren(DatagridActionOverflow) private _actionableRows: QueryList<DatagridActionOverflow>;
+    public get actionableRows(): QueryList<DatagridActionOverflow> {
+        return this._actionableRows;
+    }
```

Comment:
The logic seems off to me here, I'm having a hard time making sense of it.

You should have 2 `*ngIf`s here:
- one on the `clr-dg-cell`, which should be true if `RowActionService.actionableCount` is greater than 0. That means somewhere, one of the rows has an extra cell for actions, so they should all have the extra cell to preserve the table uniformity.
- one on the action overflow `<ng-content select="clr-dg-action-overflow">` itself, which should be true if this row _does_ have actions projected. If it doesn't, there should still be an empty cell but no ellipsis icon.